CONTRIBUTING.md

```diff
@@ -77,6 +77,24 @@ To review an open pull request submitted by another developer, there are several
 
 NOTE: We have had several branches that have exploded in commit number. If you are merging a branch and it has more than ~20 commits, strongly recommend using the "Squash and Merge" option for merging a branch.
 
+## Steps to merging a pull request
+
+To ensure consistency between our code and documentation, we need to take care of a couple of more things after accepting a review on a PR into master.
+
+1. Change the version number of the code located in `clmm/__init__.py`, commit the change to the branch, and push. If you are unsure of how you should change the version number, don't hesitate to ask!
+
+We use [semantic versioning](https://semver.org/), X.Y.Z.. If the PR makes a small change, such as a bug fix, documentation updates, style changes, etc., increment Z. If the PR adds a new feature, such as adding support for a new profile, increment Y (and reset Z to 1). If a PR adds a feature or makes a change that breaks the old API, increment X (and reset Y and Z to 1). After the first tagged release of CLMM, anything that is a candidate to increment X should be extensively discussed beforehand. 
```

Comment:
When we change a digit in the version, shouldn't we change the right digits to 0?